LabManagement/PowerShell/JITUserQuota/JITQuotaWithScheduledTask.ps1

```diff
@@ -0,0 +1,100 @@
+# Lets stop the script for any errors
+$ErrorActionPreference = "Stop"
+
+# ************************************************
+# ************ FIELDS TO UPDATE ******************
+# ************************************************
+
+# List of lab names that we should not include when updating quota)
+$excludeLabs = @('*test*','*demo*','*training*', '*how to*')
+
+# Number of available hours we reset the student to when running this script
+$usageQuota = 8
+
+# Segment of labs to update based on lab plans, regular expression to match
+
+# Match for all lab plans   
+$labPlanNameRegex = "^.*"
+
+# Subscription ID
+$subId = "1111-1111-1111-11111-11111111"
+
+# ************************************************
+
+if ($PSVersionTable.PSEdition -eq 'Desktop' -and (Get-Module -Name AzureRM -ListAvailable)) {
+    #    Write-Warning -Message ('Az module not installed. Having both the AzureRM and ' +
+    #      'Az modules installed at the same time is not supported.')
+    } else {
+        Install-Module -Name Az -AllowClobber -Scope CurrentUser -Force -Confirm:$false
+    }
+
+# Install the Az.LabServices module if the command isn't available
+if (-not (Get-Command -Name "Get-AzLabServicesLab" -ErrorAction SilentlyContinue)) {
+    Install-Module -Name Az.LabServices -Scope CurrentUser -Force
+}
+
+Import-Module Az
+Import-Module "..\BulkOperations\Az.LabServices.BulkOperations.psm1" -Force
```

Comment:
Here we assume the location of the bulk library (like it's in the repo structure), maybe we should look both in this folder and the current directory, and if it's not found alert the user that it's a dependency?